src/olympia/versions/utils.py

```diff
@@ -45,12 +46,25 @@ def write_svg_to_png(svg_content, out):
 def encode_header_image(path):
     try:
         with storage.open(path, 'rb') as image:
-            header_blob = image.read()
+            _, file_ext = os.path.splitext(path)
+            return encode_header(image.read(), file_ext)
+    except IOError as io_error:
+        log.debug(io_error)
+        return (None, 0, 0)
+
+
+def encode_header(header_blob, file_ext):
+    try:
+        if file_ext == '.svg':
+            width = int(pq(header_blob).attr('width'))
+            height = int(pq(header_blob).attr('height'))
```

Comment:
mmmmmm `pyquery` in production code makes me nervous. It's one thing to work around its quirks in unit tests, it's another to have to potentially deal with that in production... Any chance to avoid that ?

Reply:
Yeah it should be safe for our needs. Probably safer than `pyquery`, in fact.

Reply:
https://github.com/mozilla/addons-server/pull/9862/commits/ffcb1674ba5b59fc902ae7a4b32ec415fc2cecc4 (assuming travis agrees)